index.js

```diff
@@ -56,6 +56,25 @@ module.exports = {
     'import/no-named-default': 'error',
     'import/no-webpack-loader-syntax': 'error',
     'import/prefer-default-export': 'error',
+    'jsdoc/check-examples': 0,
+    'jsdoc/check-param-names': 1,
+    'jsdoc/check-tag-names': 1,
+    'jsdoc/check-types': 1,
+    'jsdoc/newline-after-description': 0,
+    'jsdoc/no-undefined-types': 0, // did not consider InputValidationError as valid type
+    'jsdoc/require-description': 0,
```

Comment:
Obviously this is opinion but why we don't require description? Even short one can be useful and if it's not really adding value like format response - it's obvious what the function does. It's not harm to simply put it there anyway

Reply:
I think in some cases (e.g. some small util) functions are very self explanatory, description will be just added noise there.

Still downside of keeping this off, is that we may have some non obvious functions poorly documented.

Also other issue, is that we will now be forced to add this in existing code, so it may happen that in large file you need to describe functions you're not sure of proper intention. To avoid confusion I think it's better to leave description off then.

Reply:
Valid points - basically opinion vs opinion :)

Should we run a poll?

> you're not sure of proper intention
I think linter will accept something like: `TODO: Add description here` if you turn on the rule here for a problematic functions

Reply:
We may also leave it off, and turn it on if we indeed observe, people avoid adding description in non obvious cases. @mfonsen  ?

Reply:
There would be whopping 1136 errors in backend to fix if we turn this on. I would consider this rule separately later. No automatic fix available.

Reply:
As I checked it actually requires using `@description` token, which I find wierd.

As we do not use it anywhere it's no wonder it'll error for very JSDoc statement.

I'm not sure what's value in marking description with `@description`, I would keep it off

Reply:
ok. Thank you for the additional details